omniwitness/logs.yaml

```diff
@@ -21,6 +21,12 @@ Logs:
     PublicKey: rekor.sigstore.dev+c0d23d6a+AjBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABNhtmPtrWm3U1eQXBogSMdGvXwBcK5AW5i0hrZLOC96l+smGNM7nwZ4QvFK/4sueRoVj//QP22Ni4Qt9DPfkWLc=
     Feeder: rekor
     UseCompact: false
+  - Origin: rekor.sigstore.dev - 1193050959916656506
```

Comment:
Are you able/would you consider shifting your origin string for new logs to the "schema-less URI" scheme described in the [tlog-checkpoint spec](https://github.com/C2SP/C2SP/blob/main/tlog-checkpoint.md#note-text)?

e.g. `rekor.sigstore.dev/1193050959916656506` might be one option.

Reply:
Unfortunately didn't get a chance to do this yesterday before the sharding. We are tracking it in https://github.com/sigstore/rekor/issues/2069. We will definitely do this with v2 tile-based Rekor.

Reply:
Sorry, wrong (but sort of related) issue, https://github.com/sigstore/rekor/issues/1450 is it.

Reply:
No worries, they can go in like this - nothing will break, and we can get them spec compliant on the next, hopefully Tessera-based, shard :)